app/screens/home/SettingScreen.tsx

```diff
@@ -620,13 +615,14 @@ const SettingScreen = (props: any): any => {
       setlanguage(allCountries[0]?.languages[0]);
     } else {
       console.log('Selected country for countryId is', countryId);
-      const selectedCountry: any = allCountries.find(
-        (country: any) => country.CountryID === countryId,
+      console.log('allCountries country for countryId is', allCountries);

```

Comment:
This message is confusing, let's please improve.

Reply:
This log added for testing purpose so removed now. thanks